Pipfile

```diff
@@ -10,9 +10,4 @@ pyyaml = "*"
 [dev-packages]
 
 [requires]
-python_version = "3.8"
-
-[packages.scratch_var]
```

Comment:
why were we using this before ?

Reply:
Scratch var is used in NFT example project when it wasn't released. So we decided to explicitly add & update pipfile then